.travis.yml

```diff
@@ -20,7 +23,8 @@ jobs:
             - g++-6
       install: *ccache-setup-steps
       script:
-        - pyenv global 2.7.15
+        - pyenv global ${PYTHON_VERSION}
+        - export PYTHONPATH="${PYTHONPATH}:$(pwd)/tools"
```

Comment:
why does PYTHONPATH have to be set? does everyone have to? `./configure` should be able to import from tools itself, without needing this. I checked out your branch and tried it, and I didn't have to set any paths to call configure. Still building to confirm.

Reply:
Also, I have python2.7 and python3.7 in /usr/bin (ubuntu), and 2.7 is chosen. Is this intended? I can't even use python3 manually:
```
core/lts (pr/28537 *$) % python3 configure
sys.version_info(major=3, minor=7, micro=3, releaselevel='final', serial=0)
Please use Python 2.7:

  /usr/bin/python2 configure
```

Reply:
Awesome...  Thanks for digging into this.  Files like __tools/configure.d/nodedownload.py__ will need to be able to __import six__ but if that is enabled then I am all good.

You are forced to use legacy Python because this PR does not include something like #25878.  Small moves...

Reply:
I'll go look at #25878